core/utils/dmljob.py

```diff
@@ -129,7 +132,7 @@ def deserialize_job(serialized_job):
         rest['uuid'],
         rest['raw_filepath'],
         rest['session_filepath'],
-        rest['datapoint_count']
+        rest['datapoint_count'],
```

Comment:
Did you mean to add `round_num` here too?

Reply:
Nope! Decided against it